lib/compiler/passes/generate-bytecode.js

```diff
@@ -395,6 +408,7 @@ function generateBytecode(ast) {
               action: null
             }),
             buildCondition(
+              elements[0].match | 0,
```

Comment:
what does this mean?

Reply:
No, it wasn't.  You gave a Zorba the Mystic "when you look deep enough into the soul of `match` you will understand."

Reply:
It turns out this is `(elements || [])[0]?.match === 0`

Admittedly, getting this idiomatically correct in a way that's clear to a reader and doesn't rely on trivia knowledge of operator behavior isn't so easy.